x-pack/filebeat/module/gsuite/_meta/fields.yml

```diff
@@ -0,0 +1,40 @@
+- key: gsuite
+  title: "gsuite"
+  description: >
+    gsuite Module
+  fields:
+    - name: gsuite
+      default_field: false
+      type: group
+      description: >
+        Gsuite specific fields.
+
+        More information about specific fields can be found at https://developers.google.com/admin-sdk/reports/v1/reference/activities/list
+      fields:
+        - name: actor.key
+          type: keyword
+          description: >
+            Only present when callerType is KEY. Can be the consumer_key of the requestor for OAuth 2LO API requests or an identifier for robot accounts.
+        - name: actor.type
+          type: keyword
+          description: >
+            The type of actor.
+
+            Values can be:
+              *USER*: Another user in the same domain.
+              *EXTERNAL_USER*: A user outside the domain.
+              *KEY*: A non-human actor.
+        - name: event.type
```

Comment:
Is there a link we could add to this? Maybe I'm just too caught up in ECS `event.type` but `gsuite.event.type` v. `event.type` would probably require me searching for some references around what values this could actually be.